.gitignore

```diff
@@ -21,4 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
-replay_pid*
+replay_pid*
```

Comment:
let's fix the EOF lines or otherwise they'll go unnoticed

Reply:
Resolved with 6c3be6a7ee78775a2fd8022a4dba2050981d4174